src/EntityFramework/Infrastructure/Interception/DatabaseLogFormatter.cs

```diff
@@ -30,6 +30,7 @@ namespace System.Data.Entity.Infrastructure.Interception
     /// </remarks>
     public class DatabaseLogFormatter : IDbCommandInterceptor, IDbConnectionInterceptor, IDbTransactionInterceptor
     {
+        private const string StopwatchStateKey = "__LoggingStopwatch__";
         private readonly WeakReference _context;
         private readonly Action<string> _writeAction;
         private readonly Stopwatch _stopwatch = new Stopwatch();
```

Comment:
@AndriySvyryd It is actually still always used to preserve back compatibility with code that has overridden some but not all methods.